process-migration/tests/features/process-migration.feature

```diff
@@ -0,0 +1,10 @@
+@rhpam-7/rhpam75-process-migration
+Feature: RHPAM Process Migration tests
+
+   Scenario: Test REST API is available and valid
+     When container is ready
+     Then check that page is served
+         | property | value |
+         | port     | 8080  |
+         | path     | /rest/health/status |
+         | expected_phrase | SUCCESS |
```

Comment:
is this test applicable to this new container?

Also, is there more tests that can be done?

Reply:
This one should be enough, it is the health probe that will be used when deployed